.storybook/webpack.config.js

```diff
@@ -1,7 +1,11 @@
 const defaultConfig = require('@stoplight/storybook-config/webpack.config');
+const path = require('path');
```

Comment:
```suggestion
const path = require('@stoplight/path');
```
?

Reply:
I don't think it's installed, yet can do.